<launch>

  <node name="hmi_io" pkg="hmi_io" type="hmi_io" args="--force-discover" >
   <param name="default_topicCam" value="/camera_me_obstacle"/>
   <param name="default_topicLidar" value="/lidar_obstacle"/>
   <param name="default_topicEnv" value="/obstacles"/>
  </node>
 
</launch>
